diff --git a/scanssh-2.1.2/configure.in b/scanssh-2.1.2/configure.in
index ecff1be..a0a0824 100644
--- a/scanssh-2.1.2/configure.in
+++ b/scanssh-2.1.2/configure.in
@@ -98,8 +98,8 @@ AC_ARG_WITH(libpcap,
      ;;
   esac ],
 [ if test -f ${prefix}/include/pcap.h; then
-     PCAPINC="-I${prefix}/include"
-     PCAPLIB="-L${prefix}/lib -lpcap"
+     PCAPINC=""
+     PCAPLIB="-lpcap"
   elif test -f /usr/include/pcap/pcap.h; then
      PCAPINC="-I/usr/include/pcap"
      PCAPLIB="-lpcap"
